doc/source/whatsnew/v0.18.1.txt

```diff
@@ -131,6 +131,7 @@ API changes
 
 
 
+- the default for ``.query()/.eval()`` is now ``engine=None`` which will use ``numexpr`` if its installed, else will fallback to the ``python`` engine. This mimics the pre-0.18.1 behavior if ``numexpr`` is installed (and which previously would raise if ``numexpr`` was NOT installed and ``.query()/.eval()`` was used). (:issue:`12749`)
```

Comment:
"its" -> "it's". Maybe change "and which would previously..." to "Previously, if ``numexpr`` was not installed, ``.query()/.eval()`` would raise."

Does it raise a PerformanceWarning when it falls back to python? Might be good to mention that here.